### DISCUSSION TITLE
Jetpack Compose

### DISCUSSION THREAD

**Discussion by mar10uane** [AndroidIDEOfficial/AndroidIDE#1582] [Ideas]
I really like to know if you have some future plans for adding **Jetpack compose** support

 Like is is possible?

**Comment by PriNova** [comment]
> I really like to know if you have some future plans for adding **Jetpack compose** support
> 
> Like is is possible?

You can use Jetpack compose right now.
Put the necessary dependencies and activate compose in your build.gradle.kt file in the app module with this:

```
android {
   
    [other parts left out for brevity]

    compileOptions {
        sourceCompatibility = JavaVersion.VERSION_1_8
        targetCompatibility = JavaVersion.VERSION_1_8
    }
    kotlinOptions {
        jvmTarget = "1.8"
    }
    buildFeatures {
        buildConfig = true
        compose = true
    }
    composeOptions {
        kotlinCompilerExtensionVersion = "1.5.7"
    }
}
```
